script.py

```diff
@@ -86,14 +75,14 @@
                 comments_box.send_keys('Good')
             except:
                 None
-                
+
         driver.execute_script("window.scroll(0, 0);")
 
         suggestion_boxes = driver.find_elements_by_class_name('suggestion_boxes')
 
         for suggestion_box in suggestion_boxes:
             try:
-                suggestion_box.send_keys(' Teacher')
+                suggestion_box.send_keys('Teacher')
```

Comment:
Give Space Before "Teacher" 
the comment will be: " Good Teacher

Reply:
Noted